packages/cli/package.json

```
@@ -46,6 +46,11 @@
     "@cross/dir": "jsr:@cross/dir@^1.1.0",
     "@hongminhee/localtunnel": "^0.3.0",
     "@std/fmt/colors": "jsr:@std/fmt@^0.224.0/colors",
-    "@david/dax": "jsr:@david/dax@^0.43.2"
+    "@david/dax": "jsr:@david/dax@^0.43.2",
+    "cli-table3": "^0.6.5",
+    "hono": "jsr:@hono/hono@^4.8.3",
```

We should avoid JSR packages in *package.json*:

```suggestion
    "hono": "^4.8.3",
```